cfbs/pretty_json.py

```diff
@@ -0,0 +1,114 @@
+import json
+from typing import OrderedDict
+
+
+def pretty(s):
+    MAX_LEN = 80
+    INDENT_SIZE = 2
+
+    def _force_wrap(parent):
+        if len(parent) < 2:
+            return False
+        for child in parent:
+            if not isinstance(child, (tuple, list, dict)):
+                return False
+            if len(child) < 2:
+                return False
```

Comment:
Count number of children with 2 or more elements. If count is more than 1, wrap.